proto/cosmos/distribution/v1beta1/tx.proto

```diff
@@ -14,25 +14,38 @@ import "cosmos/distribution/v1beta1/distribution.proto";
 service Msg {
   // SetWithdrawAddress defines a method to change the withdraw address
   // for a delegator (or validator self-delegation).
-  rpc SetWithdrawAddress(MsgSetWithdrawAddress) returns (MsgSetWithdrawAddressResponse);
+  rpc SetWithdrawAddress(MsgSetWithdrawAddress)
+      returns (MsgSetWithdrawAddressResponse);
 
   // WithdrawDelegatorReward defines a method to withdraw rewards of delegator
   // from a single validator.
-  rpc WithdrawDelegatorReward(MsgWithdrawDelegatorReward) returns (MsgWithdrawDelegatorRewardResponse);
+  rpc WithdrawDelegatorReward(MsgWithdrawDelegatorReward)
+      returns (MsgWithdrawDelegatorRewardResponse);
 
   // WithdrawValidatorCommission defines a method to withdraw the
   // full commission to the validator address.
-  rpc WithdrawValidatorCommission(MsgWithdrawValidatorCommission) returns (MsgWithdrawValidatorCommissionResponse);
+  rpc WithdrawValidatorCommission(MsgWithdrawValidatorCommission)
+      returns (MsgWithdrawValidatorCommissionResponse);
 
   // FundCommunityPool defines a method to allow an account to directly
   // fund the community pool.
-  rpc FundCommunityPool(MsgFundCommunityPool) returns (MsgFundCommunityPoolResponse);
+  rpc FundCommunityPool(MsgFundCommunityPool)
+      returns (MsgFundCommunityPoolResponse);
 
-  // UpdateParams defines a governance operation for updating the x/distribution module
-  // parameters. The authority is defined in the keeper.
+  // UpdateParams defines a governance operation for updating the x/distribution
+  // module parameters. The authority is defined in the keeper.
   //
   // Since: cosmos-sdk 0.47
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
+
+  // CommunityPoolSpend defines a governance operation for sending tokens from
+  // the community pool in the x/distribution module to another account, which
+  // could be the governance module itself. The authority is defined in the
+  // keeper.
+  //
+  // Since: cosmos-sdk 0.47
+  rpc CommunityPoolSpend(MsgCommunityPoolSpend)
```

Comment:
I know I said this new Msg was fine by me yesterday, but I'm having 2nd thoughts.

How would one do a gov proposal with an IBC transfer? We would basically need 2 Msgs in one proposal, right? It might be confusing though.

I think an authz here would actually be much easier.

Reply:
You would need a 2nd message to do the ICS20 transfer to another chain.

community pool -> gov module, OR
community pool -> regular account -> ICS20 transfer

Reply:
The altnernative is an authz, then you could do community pool -> ICS20 transfer directly. I think I prefer this tbh.

Reply:
Can you explain how that would work? Who is the grantee of the authz grant and what message(s)?

Reply:
You can't use `MsgTransfer` or `MsgSend` though, the community pool isn't actually a module account, it's a "virtual" account. Thats why I created `MsgCommunityPoolSpend`. So would authz still work here? Note, that with `MsgCommunityPoolSpend` you still can't do IBC transfers directly.

Reply:
SGTM. I'm fine with keeping this Msg then and using the gov module as an intermediary account when submitting gov proposals.